students/Hebdowska_Grazyna/lesson_08/strong_pasword.py

```diff
@@ -0,0 +1,30 @@
+import re
+
+
+def check_strong(password):
+
+    re_number1 = re.compile(r'''((\w[a-z]{1,})+)''')
+    re_number2 = re.compile(r'''((\w[A-Z]{1,})+)''')
+    re_number3 = re.compile(r'''((\d{1,})+)''')
+    mo1 = re_number1.search(password)
+    mo2 = re_number2.search(password)
+    mo3 = re_number3.search(password)
+    if mo1 is None:
+        return False
+    elif mo2 is None:
+        return False
+    elif mo3 is None:
+        return False
```

Comment:
You might join the if statements:
    if mo1 is None or mo2 is None or mo3 is None:
        return False
    return True